admin-dev/themes/new-theme/js/app/pages/permission/components/bulk.vue

```diff
@@ -0,0 +1,117 @@
+<template>
```

Comment:
should we prefer using PascalCase or kebab-case for vue files? :thinking:

Reply:
All vue files are in kebab-case in this project.